cmake/modules/BuildDPDK.cmake

```diff
@@ -176,7 +176,7 @@ function(do_export_dpdk dpdk_dir)
   set_target_properties(dpdk::dpdk PROPERTIES
     INTERFACE_INCLUDE_DIRECTORIES ${DPDK_INCLUDE_DIR}
     INTERFACE_LINK_LIBRARIES
-    "-Wl,--whole-archive $<JOIN:${DPDK_ARCHIVES}, > -Wl,--no-whole-archive")
+    "-Wl,--whole-archive $<JOIN:${DPDK_ARCHIVES}, > -Wl,--no-whole-archive ${dpdk_numa} -Wl,-lpthread,-ldl")
```

Comment:
`${dpdk_numa}` does not belong to this commit.

Reply:
${dpdk_numa} belongs to commit 58d0be5dcee41b19f8d94654c3b05177cd3b2495.
commit 98229bf69a8cb22af70504feeec448a99ee68abd only add '-Wl,-lpthread,-ldl',don't contain any numa

Reply:
ok,I split the commit

Reply:
98229bf  depends on 58d0be5.